src/org/sosy_lab/java_smt/solvers/princess/PrincessEnvironment.java

```diff
@@ -661,6 +730,15 @@ static <T> Seq<T> toSeq(List<T> list) {
     return collectionAsScalaIterableConverter(list).asScala().toSeq();
   }
 
+  static Seq<ITerm> toITermSeq(List<IExpression> exprs) {
+    return PrincessEnvironment.toSeq(
+        exprs.stream().map(e -> (ITerm) e).collect(Collectors.toList()));
+  }
+
+  static Seq<ITerm> toITermSeq(IExpression... exprs) {
+    return toITermSeq(Arrays.asList(exprs));
+  }
+
   IExpression simplify(IExpression f) {
     // TODO this method is not tested, check it!
```

Comment:
Native solver tests seem appropriate considering there is some assumptions that we put on the behavior of the solver.

Reply:
Do you mean `simplify` here?